src/clr-angular/forms/checkbox/_checkbox.clarity.scss

```diff
@@ -1,6 +1,8 @@
-// Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
-// This software is released under MIT license.
-// The full license information can be found in LICENSE in the root directory of this project.
+/*!
```

Comment:
Noticed as well that Material doesn't put copyright notices on non typescript files (since they are bundled I guess), nor on spec files.

Reply:
I've seen them on some of our scss files but not others. I thought this would be a good time to make them all the same.